hail/src/main/c/davies.cpp

```diff
@@ -344,28 +344,27 @@ real DaviesAlgo::qf(real* lb1, real* nc1, int* n1, int r1, real sigma, real c1,
 
 extern "C"
 EXPORT
-real qfWrapper(real* lb1, real* nc1, int* n1, int r1, real sigma, real c1,
+real qfWrapper(real* lb1, real* nc1, int* dof, int r1, real sigma, real c1,
```

Comment:
see the caller `Skat.scala` to understand why the transformations here are the correct ones. I think it was pedantic that actually caught all these unused variables and non-standard c.